website/pages/en/support.js

```diff
@@ -16,12 +16,12 @@ The React Native for Windows (RNW) Team strives to provide full support for the
 | Version | Support Phase | Release Date | Active Support Start | Maintenance Support Start | End of Support |
 | -- | -- | -- | -- | -- | -- |
 | [main](https://www.npmjs.com/package/react-native-windows/v/canary) | [Canary](#canary-support) | *N/A* | *N/A* | *N/A* | *N/A* |
-| [0.75](https://www.npmjs.com/package/react-native-windows/v/preview) | [Preview](#preview-support) | *TBD* | *TBD* | *TBD* | *TBD* |
-| [0.74](https://www.npmjs.com/package/react-native-windows/v/latest) | [Active](#active-support) | 4/29/2024 | 4/29/2024 | *TBD* | *TBD* |
+| [0.76](https://www.npmjs.com/package/react-native-windows/v/preview) | [Preview](#preview-support) | *TBD* | *TBD* | *TBD* | *TBD* |
```

Comment:
Have they already snapped 0.76-stable? I don't think we have a 0.76 preview yet. I'd hold off on adding 0.76 until we have an actual 0.76 preview to support.